frontend/yarn.lock

```diff
@@ -0,0 +1,8938 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I'm betting this should also be in .gitignore

Reply:
https://yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

.lock files in for npm and yarn are recommended to be committed as they tell you what was the versions of the dependencies your were using. This will track all dependencies not just the ones we declare on package.json but the "dependencies of the dependencies".

Reply:
I stand corrected - thanks for clarifying!